README.md

```diff
@@ -56,3 +58,14 @@ to reflect these differences.
 The code and our documentation are the ultimate sources of truth. CFPs are merely
 the starting point for the implementation.
 
+# Status
+
+ The status of a CFP indicates where it is in the process. The CFP process is as follows:
+
+ 1. ``Provisional``: All new PRs are given this status. This CFP's design has not yet been approved.
+ 2. ``Implementable``: Approved PRs that are merged are given this status. Their CFP design has been approved, and now the work can be started to implement them.
+ 3. ``Experimental``: The CFP has been implemented and is now in a 'Beta' state.
+ 4. ``Stable``: The CFP has been implemented, rigororously tested, and is now in a 'GA' state.
+ 5. ``Complete``: All implementation work is complete and has been merged.
+
+ At any time during the process, the CFP can be given the status of ``Withdrawn`` if it is no longer thought to be viable, or if it is not wanted to be pursued for any other reason.
```

Comment:
To help inform the thought process, I have a PR (https://github.com/cilium/design-cfps/pull/36) that sort-of fits this category. I labelled it "Dormant" for now, because in this case the design review resulted in "Changes Requested", but I do not currently have the cycles to pursue the change further. The core idea is still worthwhile, but it hasn't proven critical enough to solve just yet. The idea isn't exactly "Withdrawn", but the specific design is practically speaking "Withdrawn". I'm not sure whether I would consider that one to be under "Provisional" or "Withdrawn" given that the primary blocker on moving it forward is just interest from a contributor to resolve the outstanding feedback and push it forward.